change/@fluentui-react-provider-8a587528-2d75-439f-9301-2505033412fb.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "refactor: rename main DrawerOverlay/DrawerInline components",
```

can we improve this ? its a bit confusing from changelog POV what change does this mean.

```suggestion
  "comment": "feat: add new drawer global style hooks that are needed by drawer public api changes",
```